src/SourceBuild/content/eng/build.targets

```diff
@@ -0,0 +1,29 @@
+<Project>
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="FindArchiveDiffs" />
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="GetClosestOfficialSdk" />
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="GetSingleArchiveItem" />
+
+  <Target Name="ReportSdkArchiveDiffs"
+          AfterTargets="Build"
+          DependsOnTargets="DetermineSourceBuiltSdkVersion" >
+
+      <GetSingleArchiveItem SdkArchiveItems="@(SdkTarballItem)">
+        <Output TaskParameter="BestSdkArchiveItem" PropertyName="_BuiltSdkArchivePath"/>
+      </GetSingleArchiveItem>
+
+      <GetClosestOfficialSdk BuiltSdkPath="$(_BuiltSdkArchivePath)" Condition="'$(_BuiltSdkArchivePath)' != ''">
+        <Output TaskParameter="ClosestOfficialSdkPath" PropertyName="ClosestOfficialSdkPath" />
+      </GetClosestOfficialSdk>
+
+      <Message Text="Failed to find closest official SDK archive." Importance="High" Condition="'$(_BuiltSdkArchivePath)' != '' AND '$(ClosestOfficialSdkPath)' == ''" />
```

Comment:
If the built archive path was empty, wouldn't this also be an error?

Reply:
Yeah, I reworked some of the errors. We should now error in the tasks if we can't find a valid archive, or if downloading the closest official sdk fails.